frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -165,11 +138,12 @@ export const surveyLogic = kea<surveyLogicType>([
         updateSurveySuccess: async ({ survey }) => {
             lemonToast.success(<>Survey {survey.name} updated</>)
             actions.editingSurvey(false)
-            router.actions.replace(urls.survey(survey.id))
```

Comment:
Not familiar with issue here but is the route not necessary anymore?

Reply:
I noticed it doesn't seem to do anything 🤔